src/include/stir/ProjData.h

```diff
@@ -301,7 +301,9 @@ class ProjData : public ExamData
   inline int get_num_sinograms() const;
   //! Get the total size of the data
   inline std::size_t size_all() const;
-  
+  //! writes info to a file in Interfile format
```

Comment:
I think `info` is a bit confusing as it sounds like it's only `ProjDataInfo`. Say `data` or so?

Reply:
:ok: